app/src/pages/admin/projectsPage/projectMenu/projectMenu.jsx

```diff
@@ -0,0 +1,185 @@
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { injectIntl, defineMessages, intlShape } from 'react-intl';
+import { redirect } from 'redux-first-router';
+
+import {
+  assignToProjectAction,
+  unassignFromProjectAction,
+  assignedProjectsSelector,
+  userIdSelector,
+} from 'controllers/user';
+import { showModalAction } from 'controllers/modal';
+import { deleteProjectAction } from 'controllers/administrate/projects';
+import { PROJECT_DETAILS_PAGE } from 'controllers/pages';
+import { SETTINGS, MEMBERS } from 'common/constants/projectSections';
+import { DotsMenuButton, SEPARATOR_ITEM, DANGER_ITEM } from 'components/buttons/dotsMenuButton';
+import { ADMIN_ALL_PROJECTS_PAGE_MODAL_EVENTS } from 'components/main/analytics/events';
+
+const messages = defineMessages({
+  members: {
+    id: 'ProjectPanel.members',
+    defaultMessage: 'Members',
+  },
+  settings: {
+    id: 'ProjectPanel.settings',
+    defaultMessage: 'Settings',
+  },
+  assign: {
+    id: 'ProjectPanel.assign',
+    defaultMessage: 'Assign',
+  },
+  unassign: {
+    id: 'ProjectPanel.unassign',
+    defaultMessage: 'Unassign',
+  },
+  delete: {
+    id: 'ProjectPanel.delete',
+    defaultMessage: 'Delete',
+  },
+  unassignFromPersonal: {
+    id: 'ProjectPanel.unassignFromPersonal',
+    defaultMessage: 'Impossible to unassign user from personal project',
+  },
+  modalCancelButtonText: {
+    id: 'ProjectsPage.modal.modalCancelButtonText',
+    defaultMessage: 'Cancel',
+  },
+  deleteModalTitle: {
+    id: 'ProjectsPage.modal.deleteModalTitle',
+    defaultMessage: 'Delete project',
+  },
+  deleteModalConfirmationText: {
+    id: 'ProjectsPage.modal.deleteModalConfirmationText',
+    defaultMessage: 'Are you sure you want to delete the project <b>{name}</b>?',
+  },
+  deleteModalSubmitButtonText: {
+    id: 'ProjectsPage.modal.deleteModalSubmitButtonText',
+    defaultMessage: 'Delete',
+  },
+  deleteError: {
+    id: 'ProjectsPage.deleteError',
+    defaultMessage: 'An error occurred during deleting the project',
```

Comment:
I have the same messages for the bulk `delete` operation in Grid view in `projectsPage/projectsToolbar/projectsToolbar.jsx`, maybe we can use a shared mesage config for them?

Reply:
Is it a good idea to extract all messages into separate file for common use in Projects?

Reply:
I think it's a good idea

Reply:
Done!